src/main/java/org/elasticsearch/search/aggregations/reducers/Reducer.java

```diff
@@ -66,6 +69,13 @@
 
     }
 
+    public static final Function<Aggregation, InternalAggregation> FUNCTION = new Function<Aggregation, InternalAggregation>() {
```

Comment:
not in this PR, but in this file... can you change `ParseField("bucketsPath")` to `ParseField("buckets_path")`

please don't use camelCasing